cmd/getConfig.go

```diff
@@ -93,38 +92,60 @@ func getDockerImageValue(stepName string) (string, error) {
 	return dockerImageValue, nil
 }
 
-func getConfig() (config.StepConfig, error) {
-	var myConfig config.Config
-	var stepConfig config.StepConfig
+func getBuildTooFromStageConfig(stepName string) (string, error) {
```

Comment:
```suggestion
func getBuildToolFromStageConfig(stepName string) (string, error) {
```

Found another one :laughing:

Reply:
Shame ... 😁